pkg/apb/executor.go

```diff
@@ -198,6 +198,8 @@ func createExtraVars(context *Context, parameters *Parameters) (string, error) {
 	if context != nil {
 		paramsCopy["namespace"] = context.Namespace
 	}
+
+	paramsCopy["cluster"] = runtime.Provider.GetRuntime()
```

Comment:
I'm not sure if I believe this should be an extra variable or an environment variable that APBs should load and have default values for...

Reply:
I'm curious the reason that you have this as an argument instead of an environment variable to the APB.

Reply:
+1 for curiosity David. The reason I went with extravars is because it will overwrite the value of ```cluster``` right from the command line.  Then we don't have to 1) add a default environment var to apb-base, 2) I don't need to rely on the user setting CLUSTER=kubernetes to run the playbooks locally, and 3) I can default the var right in the playbook:
```yaml
vars:
     cluster: kubernetes
```
```
docker run docker.io/ansibleplaybookbundle/mediawiki-apb provision --extra-vars cluster=kubernetes

ansible-playbook playbooks/provision.yaml --extra-vars "cluster=kubernetes"
```